HygrometerApp/Resources/Network/API.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  API.swift
+//  HygrometerApp
+//
+//  Created by hyeonseok on 2022/07/11.
+//
+
+import Foundation
+
+public struct API {
+    public init() {}
```

Comment:
혹시 여기 `public`으로 쓰신 거 오타나신걸까요?
모든 메소드가 타입 메서드로 되어있어서 아래와 같이 바꾸는 게 코드 문맥 상 맞는 거 같아요.
```suggestion
    private init() {}
```

Reply:
음 재사용을 위해서 하긴한건데. 그럼 API 당 별도로 다 사용해야겠네요?

Reply:
모든 메서드가 타입 메서드인데, 나중에 일반 메서드를 구현할 생각인가요? 이에 대해 PR에 작성해주시면 좋겠어요 ㅜ